README.md

```diff
@@ -40,6 +40,17 @@ We have chosen the Rust programming language for Bee because of its C/C++ like p
 
 :::
 
+## Development
+
+The Bee repository has different branches:
+
+|Branch|Description|
+|------|-----------|
+|`production`|The latest release for the IOTA networks (`mainnet`, `devnet`, `comnet`).|
+|`develop`|The ongoing development for future releases of these networks. With every release, the `develop` branch will be merged into `production`.|
+|`staging`|The latest release for the Shimmer networks (`testnet`).|
```

Comment:
Same, we may add/remove networks and we would always have to remember about that.

Reply:
```suggestion
|`staging`|The latest release for the Shimmer networks.|
```